OcelotApiGateway/ocelot.global.json

```diff
@@ -278,16 +278,42 @@
       "UpstreamPathTemplate": "/api/schedule/examination/schedule-emergency",
       "UpstreamHttpMethod": [ "Get", "Post" ]
     },
+<<<<<<< HEAD
+
+    {
+      "DownstreamPathTemplate": "/schedule/examination/schedule-renovation",
+=======
     {
       "DownstreamPathTemplate": "/schedule/examination/by-specialty/{everything}",
+>>>>>>> master
+      "DownstreamScheme": "http",
+      "DownstreamHostAndPorts": [
+        {
+          "Host": "$PSW_SCHEDULE_SERVICE_HOST",
+<<<<<<< HEAD
+          "Port": "$PSW_SCHEDULE_SERVICE_PORT"
```

Comment:
Resolve these conflicts and remove quotes for port env var.

Reply:
Resolved.